modules/floating-buttons/widgets/floating-bars-var-1.php

```diff
@@ -29,6 +29,10 @@ public function get_group_name(): string {
 		return 'floating-bars';
 	}
 
+	public function get_style_depends(): array {
+		return [ 'widget-floating-buttons' ];
```

Comment:
Maybe you'll need to add a check for `load_widgets_styles_in_head()`. Keep it in mind.